cv32e40p/sim/tools/README.md

```diff
@@ -1,7 +1,21 @@
 ## TOOLS directory
-Tool-specific sub-dirs for various tools used in the CV32 project.  Please do
-**not** place these in the sim, sim/core or sim/uvmt\_cv32 directories.
+Tool-specific sub-dirs for various tools used in the CV32E40P project.
+Please do **not** place these in the sim, sim/core or sim/uvmt directories.
+
+#### dsim
+Code coverage scopes configuration for Metrics dsim.
+
+#### gtk
+Handy configs for bringing up top-level signals on the gtkwave waveform viewer.
+
+#### imperas
 
 #### mcy
 Setup and configuration for [Mutual Coverage with Yosys](https://github.com/YosysHQ/mcy) from [SymbioticEDA](https://www.symbioticeda.com/).
```

Comment:
Seems like an old typo? It should be called _"Mutation Cover with Yosys"_?

https://github.com/YosysHQ/mcy

Reply:
I've always through "Mutation Cover with Yosys" was bad English.  :-)